<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelOfGame</class>
 <widget class="QMainWindow" name="LevelOfGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="easyButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pristina</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>easy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="veryEasyButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pristina</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>very easy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mediumButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pristina</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>medium</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hardButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pristina</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>hard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="veryHardButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pristina</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>very hard</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>75</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>75</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Pristina</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose the level of game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>easyButton</sender>
   <signal>clicked()</signal>
   <receiver>LevelOfGame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
